server/routes/index.mjs

```diff
@@ -48,12 +56,44 @@ const getDomains = asyncMiddleware.wrap(async (req, res) => {
   }
 
   const domains = await Promise.all(companies.map(async (company) => {
-    const result = await searchCompany(company);
-    const { domain, tld } = parseDomain(result.link);
-    return {
+    const foundCompanies = await searchCompanies(company);
```

Comment:
aren't this the search results, which are not necessary the company domain 🤔 or we would like to define a search result as a company hence `foundCompanies`? I'm just saying this, because, at first sight, it seemed a little bit confusing since we have `companies` (that are extracted from the request) and `foundCompanies` for each company of these `companies` 😄